packages/rulesets/src/spectral/functions/avoid-additional-properties.ts

```diff
@@ -0,0 +1,18 @@
+const avoidAdditionalProperties = (schema: any, _opts: any, paths: any) => {
+  if (schema === null) {
+    return []
+  }
+  const path = paths.path || {}
+  if ("tags" === path[path.length - 1] && "additionalProperties" in schema) {
+    return
+  } else if ("tags" !== path[path.length - 1] && "additionalProperties" in schema) {
+    return [
+      {
+        message: "The use of additionalProperties is not allowed except for user defined tags on tracked resources.",
```

Comment:
>ot allowed except for user defined tags on tracked resources." [](http://example.com/codeflow?start=54&length=62)

The error message and documentation needs to change to reflect the change in logic